Tools/WebKitTestRunner/cocoa/TestRunnerWKWebView.mm

```diff
@@ -119,6 +118,7 @@ - (instancetype)initWithFrame:(CGRect)frame configuration:(WKWebViewConfiguratio
         [center addObserver:self selector:@selector(_didDismissPopover) name:@"UIPopoverControllerDidDismissPopoverNotification" object:nil];
         self.UIDelegate = self;
         self._inputDelegate = self;
+        self.focusStartsInputSessionPolicy = _WKFocusStartsInputSessionPolicyAuto;
```

Comment:
Do we need to reset this in `TestController::platformResetStateToConsistentValues`?

Reply:
Hm...I think the change in https://github.com/WebKit/WebKit/pull/2864/files#diff-0a820bf40aebcff03905d48e022f8c7c31443aeec49d04bbcefb5d2e32818488R273 should cover this?

Reply:
Was tripped up by the UI, which was showing this underneath `restorePortraitOrientationIfNeeded`. Looks good!